WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -151,6 +151,13 @@ class AccountSettingsService {
         return notificationObserver.map(getSettings).startWith(getSettings())
     }
 
+    func primarySiteNameForSettings(settings: AccountSettings?) -> String {
+        let service = BlogService(managedObjectContext: context)
+        let blog = service.blogByBlogId(settings?.primarySiteID)
+        
+        return blog?.settings?.name ?? String()
```

Comment:
> Regarding the String() over "", it's a personal decision: i'd rather being explicit. Literal vs new object instantiation

Definitely a personal preference, but IMO it doesn't get more explicit than `""` to say "empty string". When I see that, it's like using `Int()` to say `0`.